.github/workflows/e2e-windows.yaml

```diff
@@ -165,6 +165,14 @@ jobs:
         # Check logs for the x session
         podman logs x-session
 
+    - name: Ensure getting current HEAD version of the test framework
+      working-directory: ./extension-minikube
+      run: |
+        # workaround for https://github.com/containers/podman-desktop-extension-bootc/issues/712
```

Comment:
This wont work. You are executing it on the gh runner, not in the remote host. This code is already part of pde2e-runner: https://github.com/odockal/pde2e-runner/blob/main/lib/windows/runner.ps1#L599

Reply:
So you should be using right (latest) npm package version.